pkg/installer/utils.go

```diff
@@ -54,9 +54,6 @@ func createDirAndFiles(fs filesys.FileSystem, fsData fsData) (filesys.FileSystem
 				return fs, errors.WithStack(err)
 			}
 			for name, data := range files {
-				if _, err := fs.Create(filepath.Join(dir, subDir, name)); err != nil {
```

Comment:
I'm assuming we're just removing `Create` because it's not required as `WriteFile` creates the file if not presesnt?